eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/MockableResourceProvider.cs

```diff
@@ -70,7 +70,7 @@ private static Dictionary<InputClient, RestClientInfo> BuildRestClientInfos(IRea
                     // TODO -- here we either need to use the corresponding ResourceType.Namespace or ProviderConstants.DefaultProviderNamespace, pending on https://github.com/Azure/azure-sdk-for-net/issues/50593
```

Comment:
I think we could remove this comment since it has been implemented.

Reply:
yes, just did